ckan/templates-midnight-blue/package/snippets/package_wrench.html

```diff
@@ -1,36 +1,26 @@
-    <div class="accordion" >
-      <div class="accordion-item">
-        <h2 class="accordion-header" id="headingPackagewrench">
-            <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsedtasetwrench" aria-expanded="true" aria-controls="collapsedtasetwrench">
-              {{ _("Dataset") }}
-            </button>
-        </h2>
-
-        <div id="collapsedtasetwrench" class="accordion-collapse collapse show" aria-labelledby="headingPackagewrench" >
-            <div class="accordion-body">
-                <ul class="list-group">
-                    <li class="list-group-item d-flex justify-content-between position-relative">
-                        <a class="flex-fill" href="{{ url }}" aria-label="{{ _('Navigate to package: {name}').format(name=pkg.title or pkg.name) }}">{{ pkg.title | truncate(25)  }}</a>
-                        <div class="dropdown position-absolute end-0 me-2">
-                          <button class="btn btn-secondary btn-sm dropdown-toggle" type="button" id="dropdownRes{{ pkg.id }}" data-bs-toggle="dropdown" aria-expanded="false"><i class="fa fa-wrench"></i></button>
-                          <ul class="dropdown-menu" aria-labelledby="dropdownRes{{ pkg.id }}">
-                            {% set can_edit = h.check_access('package_update', {'id':pkg.id }) %}
-                            {% if can_edit %}
-                            <li>{% link_for _('Edit Package'), named_route=pkg.type ~ '.edit', id=pkg.name, class_='dropdown-item', icon='pencil' %}</li>
-                            {% block resources_list_edit_dropdown_inner scoped %}{% endblock %}
-                            {% endif %}
-                            <li>{% link_for _('View Package'), named_route=pkg.type ~ '.read', id=pkg.name, class_='dropdown-item', icon='chart-bar' %}</li>
-                          </ul>
-                        </div>
-                      </li>
-                </ul>
-            </div>
+  <div class="module-group">
+    <h2 class="module-heading" id="headingPackagewrench">
+          {{ _("Dataset") }}
+    </h2>
+    <ul class="list-group">
+      <li class="list-group-item d-flex justify-content-between position-relative">
+        <a {% if can_edit %}class="flex-fill"{% endif %} href="{{ pkg_url }}" aria-label="{{ _('Navigate to package: {name}').format(name=pkg.title or pkg.name) }}">{{ pkg.title | truncate(25)  }}</a>
+        {% if can_edit %}
+        <div class="dropdown position-absolute end-0 me-2">
+          <button class="btn btn-secondary btn-sm dropdown-toggle" type="button" id="dropdownRes{{ pkg.id }}" data-bs-toggle="dropdown" aria-expanded="false"><i class="fa fa-wrench"></i></button>
+          <ul class="dropdown-menu" aria-labelledby="dropdownRes{{ pkg.id }}">
+            <li>{% link_for _('Edit Package'), named_route=pkg.type ~ '.edit', id=pkg.name, class_='dropdown-item', icon='pencil' %}</li>            
```

Comment:
The consistent wrench/button UI to jump to editing the dataset and resource metadata is great.

Please use the `h.humanize_entity_type('package', dataset_type, 'edit label') or _('Edit Dataset')` thing for this button text as well.